### DISCUSSION TITLE
OAuth Support

### DISCUSSION THREAD

**Discussion by mahamudml** [kinngh/shopify-node-express-mongodb-app#27] [Q&A]
Does the repo supports the new OAuth

**Comment by kinngh** [comment]
@mahamudml can you please be a little more specific about what you're looking for with the "new" OAuth?

**Reply by mahamudml** [reply]
https://shopify.dev/apps/auth/oauth/update

**Reply by kinngh** [reply]
There's a rewrite going on for that and I'll push it over the weekend once I've tested everything^

**Reply by mahamudml** [reply]
Thanks for your efforts @kinngh
